### ISSUE TITLE
paid_via, payment mode switch from v1

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-server#7292]
For some reason, when new version was created, a behaviour from legacy site was changed. paid_via, payment_mode were switched with each other. And this is causing inconsistencies in backend and frontend. - [ ] Discuss and standardize the usage of these columns, 
- [ ] create a migration to switch between the two to make it consistent, 
- [ ] and fix server and frontend code to use correct fields where approppriate

**Comment by sp35** [comment]
@iamareebjamal Going by the example it makes sense to see `payment_mode` as `card` and `paid_via` as `stripe`; standardizing `paid_via` as the provider and `payment_mode` as the method.

**Comment by sp35** [comment]
https://github.com/fossasia/open-event-server/blob/2a55a5ff1a94d5ef19d9f1249c39d99080128b86/app/api/schema/event_invoices.py#L30-L36
https://github.com/fossasia/open-event-server/blob/2a55a5ff1a94d5ef19d9f1249c39d99080128b86/app/api/schema/orders.py#L64-L81

**Comment by iamareebjamal** [comment]
Yes, that makes sense but I'd like to discuss this in meeting once as well. I forgot last time. However,  if you want to work on it, you can start because the above reason looks the most logical. If we need to switch, it won't take much time

**Comment by faisalkhandholpur** [comment]
Please assign this issue

**Comment by faisalkhandholpur** [comment]
But why
